charts/umbrella/docs/release-notes/R24.05.md

```diff
@@ -0,0 +1,21 @@
+# Release Notes - R24.05
```

Comment:
The intention of the original [Note from R24.05 onwards](https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-2.0.11/charts/umbrella/README.md#note-from-r2405-onwards) was not to start maintaining release notes (which if we want to do that should do in a CHANGELOG.md, see [according TRG](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-3/)), but as a special note for the release due to the issue situation, which should be referred if anything as lowlight ;) because, besides other things, it made the maintenance of a FOSS sandbox much more difficult.
I recommend moving my original into a note-r2405.md file in a /docs/user/note-r2405-onwards/ directory.

Reply:
Here I am not sure if I changed it correctly, please have a look after the next commit

Reply:
I made a new comment.